From 9b1d71598b73c3b56e6db092361dc4ccef6db0f3 Mon Sep 17 00:00:00 2001
From: Hake Huang <b20222@freescale.com>
Date: Fri, 23 Mar 2012 10:55:55 +0800
Subject: [PATCH 1211/2463] ENGR00177737-1 change the drv name to sii902x_hdmi

so that system can use 2 type of HDMI device driver sii902x and on chip one

Signed-off-by: Hake Huang <b20222@freescale.com>
---
 drivers/video/mxc/mxcfb_sii902x.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/video/mxc/mxcfb_sii902x.c b/drivers/video/mxc/mxcfb_sii902x.c
index f917674..b71afc6 100644
--- a/drivers/video/mxc/mxcfb_sii902x.c
+++ b/drivers/video/mxc/mxcfb_sii902x.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2010-2011 Freescale Semiconductor, Inc. All Rights Reserved.
+ * Copyright (C) 2010-2012 Freescale Semiconductor, Inc. All Rights Reserved.
  */
 
 /*
@@ -53,7 +53,7 @@
 #include <mach/mxc_edid.h>
 #include "mxc_dispdrv.h"
 
-#define DISPDRV_SII	"hdmi"
+#define DISPDRV_SII	"sii902x_hdmi"
 
 #define TPI_PIX_CLK_LSB                 (0x00)
 #define TPI_PIX_CLK_MSB                 (0x01)
-- 
1.7.7.4

